client/scripts/precommitLint.js

```diff
@@ -26,12 +26,12 @@ sgf((err, results) => {
 
   const eslintFiles = [];
   for (const item of results) {
-    const { filename } = item;
+    const { filename, status } = item;
 
     // only include valid, filtered extensions
     // this is primarily to keep eslint rampaging
     // over non-source files
-    if (!matchesExtension(extensions, filename)) {
+    if (!matchesExtension(extensions, filename) || status === "Deleted") {
```

Comment:
Thanks for adding this, I vaguely remember adding something similar somewhere else, but maybe I missed this spot?

Reply:
Yeah, I think it was in the Node 18 branch, so I pulled it in separately here when I ran into the issue since that's not in `develop` now. We could open up a small PR with only these updates into `develop` in the meantime, or else just get them in again with DSA.